Portfolio/index.html

```diff
@@ -8,6 +8,59 @@
 
   <body style="margin: 0; padding: 0">
     <!-- Write your code below for PORTFOLIO -->
+   <h1>Chauhan.</h1>
+    <ul>
+      <li>ABOUT</li>
+      <li>SKILLS</li>
+      <li>PROJECTS</li>
+      <LI>REVIEWS</LI>
+    </ul><br>
+  <img src="/phone.icon.png" alt="Dialer Logo"/><br>
+  <button>CONTACT ME</button><br>
```

Comment:
The closing tag on line 12 should be lowercase '/h1' instead of uppercase '/h1'.